webapp/channels/src/components/sidebar/sidebar_channel/sidebar_base_channel/sidebar_base_channel_icon.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import type {ChannelType} from '@mattermost/types/channels';
+
+import SharedChannelIndicator from 'components/shared_channel_indicator';
+
+import Constants from 'utils/constants';
+
+type Props = {
+    isSharedChannel: boolean;
+    channelType: ChannelType;
+}
+
+const SidebarBaseChannelIcon = ({
+    isSharedChannel,
+    channelType,
+}: Props) => {
+    if (isSharedChannel) {
+        return (
+            <SharedChannelIndicator
+                className='icon'
+                channelType={channelType}
+                withTooltip={true}
+            />
+        );
+    } else if (channelType === Constants.OPEN_CHANNEL) {
+        return (
+            <i className='icon icon-globe'/>
+        );
+    } else if (channelType === Constants.PRIVATE_CHANNEL) {
+        return (
+            <i className='icon icon-lock-outline'/>
+        );
+    }
```

Comment:
Nit: Since we are returning inside the if, there are no need for elses.

```suggestion
    }
    if (channelType === Constants.OPEN_CHANNEL) {
        return (
            <i className='icon icon-globe'/>
        );
    }
    if (channelType === Constants.PRIVATE_CHANNEL) {
        return (
            <i className='icon icon-lock-outline'/>
        );
    }
```